homeassistant/components/freebox/router.py

```diff
@@ -31,6 +31,16 @@
 SCAN_INTERVAL = timedelta(seconds=30)
 
 
+async def get_api(hass: HomeAssistantType, host: str) -> Freepybox:
```

Comment:
Why did we move the function?

Reply:
I think we see stateless function better at top of the file

Reply:
Ok. Please do clean ups in a separate PR. 👍